fundamentals/minimal-apis/samples/MinApiTestsSample/MinApiTests/IntegrationTests/AuthorizedEndpointsIntegrationTests.cs

```diff
@@ -21,6 +21,7 @@ public AuthorizedEndpointsIntegrationTests(TestWebApplicationFactory<Program> fa
         new object[] { "true", HttpStatusCode.OK }
     };
 
+    // Verify authorized requests can access /admin endpoint.
```

Comment:
Is there a standard way to comment test code besides the good name you use for the method? I think the article will only point to this code.

Reply:
I'm not sure about any comment convention other than using XML comments for a test, only if the name doesn't
justify its use case. As for the naming convention, I feel like using `_` between words can make the test name easier ti read. Then again, it increases the character count. I don't have any opinion here.